sklearn/decomposition/_kernel_pca.py

```diff
@@ -72,6 +106,13 @@ class KernelPCA(TransformerMixin, BaseEstimator):
         Maximum number of iterations for arpack.
         If None, optimal value will be chosen by arpack.
 
+    iterated_power : int >= 0, or 'auto', default='auto'
+        Number of iterations for the power method computed by
+        svd_solver == 'randomized'. When 'auto', it is set to 7 when
+        `n_components < 0.1 * min(X.shape)`, other it is set to 4.
```

Comment:
Actually this is weird. Shouldn't we do the opposite? That is more power iterations when `n_components` is large?

Reply:
I copied the behaviour from `randomized_svd` (that is used in `PCA`) without challenging the default behaviour. Maybe we should challenge this. What about doing it in another PR though ?